Source/Core/VideoCommon/TextureCacheBase.cpp

```diff
@@ -631,6 +635,14 @@ TextureCache::TCacheEntryBase* TextureCache::Load(const u32 stage)
 			CheckTempSize(l.data_size);
 			memcpy(temp, l.data, l.data_size);
 			entry->Load(l.width, l.height, l.width, level);
+
+			// We still need to calculate the original size of the mips
+			const u32 mip_width = CalculateLevelSize(width, level);
+			const u32 mip_height = CalculateLevelSize(height, level);
+			const u32 expanded_mip_width = ROUND_UP(mip_width, bsw);
+			const u32 expanded_mip_height = ROUND_UP(mip_height, bsh);
+
+			mips_size += TexDecoder_GetTextureSizeInBytes(expanded_mip_width, expanded_mip_height, texformat);
```

Comment:
No, it's only the size of the additional mips. If there is only one level this will be zero.